src/io.jl

```diff
@@ -263,7 +281,7 @@ function record_examples(
         1
     end
     @info("starting from index $start")
-    AbstractPlotting.set_theme!(resolution = resolution)
+    setresolution!(resolution)
```

Comment:
This is a bit confusing... I find set theme clearer here, since we also reset the whole theme back to it's defaults...

Reply:
Makes sense - I've changed the function name.